corehq/apps/sso/utils.py

```diff
@@ -23,3 +26,39 @@ def get_request_data(request):
         'get_data': request.GET.copy(),
         'post_data': request.POST.copy(),
     }
+
+
+def _get_full_sso_url(view_name, identity_provider):
+    return "{}{}".format(
+        get_url_base(),
+        reverse(view_name, args=(identity_provider.slug,))
+    )
+
+
+def get_saml_entity_id(identity_provider):
+    return _get_full_sso_url("sso_saml_metadata", identity_provider)
+
+
+def get_saml_acs_url(identity_provider):
+    return _get_full_sso_url("sso_saml_acs", identity_provider)
+
+
+def get_saml_sls_url(identity_provider):
+    return _get_full_sso_url("sso_saml_sls", identity_provider)
+
+
+def get_dashboard_link(identity_provider):
```

Comment:
it's definitely still used! I hear you on later deleted code bit, but I don't believe that's the case here. I think github collapsed the forms.py diff in the summary view because it was quite large.

Reply:
Yes, that explains it.